fix(curriculum): make logical AND step more concise (#49495)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md

return "No";
```

> will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:

Author: how to attempt changes should i create new repo, if not how to do changes in a previous repo

> will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:

Author: btw i think  i`ve fixed it kindly check NVM above question

> will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:

Reviewer: This still does not use the requested wording.

> will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:

Reviewer: This still dose not use the requested wording

```
This code will return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written with the <defn>logical and</defn> operator.
```

> This will only return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written as:

Reviewer: ```suggestion
This code will return `Yes` if `num` is greater than `5` and less than `10`. The same logic can be written with the logical and operator.
```

Also, line 14 needs to be updated.
